doc/development.md

```diff
@@ -26,6 +26,8 @@ export DB_USER=...
 export DB_PASSWORD=...
 ```
 
+Note: these can be put in a `.env` file so that all variables are assigned.
```

Comment:
Let's explicitly state that the `.env` file is gitignored and should not be committed! It's a huge pain when secrets get committed :)